Description: Patch to remove dependency on "bazel_skylib"
 Currently causing build failures in bazel-bootstrap. Full bazel package will
 depend on bazel-skylib package, avoiding build failures.
Author: Jesse Chan <jc@linux.com>
Forwarded: not-needed
Last-Update: 2021-05-29

--- a/src/conditions/BUILD
+++ b/src/conditions/BUILD
@@ -1,5 +1,3 @@
-load("@bazel_skylib//lib:selects.bzl", "selects")
-
 filegroup(
     name = "srcs",
     srcs = glob(["**"]),
@@ -103,7 +101,7 @@
 )
 
 config_setting(
-    name = "darwin_arm64_constraint",
+    name = "darwin_arm64",
     constraint_values = [
         "@platforms//os:macos",
         "@platforms//cpu:arm64",
@@ -111,24 +109,6 @@
 )
 
 config_setting(
-    name = "darwin_arm64_flag",
-    values = {"cpu": "darwin_arm64"},
-)
-
-# Workaround for an issue where the dummy cc toolchain isn't being picked up
-# when cross-compile from darwin_x86_64 to darwin_arm64 cpu.
-# TODO(https://github.com/bazelbuild/bazel/issues/12655): Remove the flag based
-# select when the issue is resolved.
-selects.config_setting_group(
-    name = "darwin_arm64",
-    match_any = [
-        ":darwin_arm64_constraint",
-        ":darwin_arm64_flag",
-    ],
-    visibility = ["//visibility:public"],
-)
-
-config_setting(
     name = "darwin_arm64e",
     constraint_values = [
         "@platforms//os:macos",
